test/webserver.js

```diff
@@ -85,6 +85,12 @@ WebServer.prototype = {
       // `/../../../../../../../etc/passwd`, which let you make GET requests
       // for files outside of `this.root`.
       var pathPart = path.normalize(decodeURI(urlParts[1]));
+      // path.normalize returns a path on the basis of the current platform
+      // Windows paths were causing the statFile callback (:140)
+      // and serverDirectoryIndex function (:200) to not function correctly
+      // In the interest of avoiding platform checks at every occurence
+      // it was simpler to convert it to a unix-style path
```

Comment:
Without proper punctuation, these sentences becomes *very* difficult read.

Also, please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Reply:
I will squash the commits. Sorry about that. I'll keep that in mind from now on.

Reply:
Please use complete sentences, i.e. ones ending with a period, since that's what made for difficult to read comments in the first place!

Reply:
I have merged the commits together. Also, added correct punctuation in the comments.